### ISSUE TITLE
Consider roaring bitmaps

### ISSUE THREAD

**Issue by staltz** [ssb-ngi-pointer/jitdb#116]
They seem faster than TypedFastBitSet (according to TypedFastBitSet's own benchmarks) but `roaring` is a native node module (probably won't run in the browser).

- https://www.npmjs.com/package/roaring

Also see https://www.npmjs.com/package/bitfield-db by substack

**Comment by arj03** [comment]
Yes exactly, the non-browser part is why I didn't want to use them. Also it seems that the bitmap operation is not the part that takes up most of the time.

**Comment by staltz** [comment]
I thought so, I just wanted to track the comments I got here https://twitter.com/megaflop/status/1358771343047671810

**Comment by Barbarrosa** [comment]
@arj03 @staltz Looks like there's a WebAssembly port, i.e. [`roaring-wasm`](https://www.npmjs.com/package/roaring-wasm). I'm not sure how usable that is in the browser, but it may be worth a look.
